Source/Particles/ElementaryProcess/QEDPhotonEmission.H

```diff
@@ -96,6 +96,17 @@ public:
     * @param[in] opt_depth_functor functor to re-initialize the optical depth of the source particles
     * @param[in] opt_depth_runtime_comp index of the optical depth component of the source species
     * @param[in] emission_functor functor to generate photons and update momentum of the source particles
+    * @param[in] a_pti particle iterator to iterate over electrons or positrons undergoing QED photon emission process
+    * @param[in] lev   the mesh-refinement level
+    * @param[in] ngE   ...?
```

Comment:
@EZoni or @lucafedeli88 maybe? :)
```suggestion
    * @param[in] ngE   ...?
```

Reply:
Same as before. I'm not familiar with this module, but usually `ngE` is the number of guard cells allocated for the E and B MultiFabs. So one suggestion could be the following:
```suggestion
    * @param[in] ngE   number of guard cells allocated for the E and B MultiFabs
```
Let's see if @lucafedeli88 confirms that this is indeed the case for this specific function.

Reply:
Yes, this is called with `ngE` equal to `Ex.nGrowVect()` so what @EZoni suggested is correct.

Reply:
Thanks for clarifying. I think the question is if `ngE` is also used for the magnetic field passed into this.

Reply:
Thanks!
@NeilZaim I think the question is if `ngE` is also used for the magnetic field passed into this?

Reply:
Yes, eventually `ngE` is needed to properly compute the indices inside the field gathering done within this QED function, for both the E and B fields.

More generally, I think that the number of allocated guard cells is always the same for both the E and B MultiFabs, and is quite often called `ngE`. See for instance

https://github.com/ECP-WarpX/WarpX/blob/da1897f55e9bb4627539658fa9ba743c507fb287/Source/WarpX.cpp#L1599-L1605

If we think that this is not explicit enough, then we can try to use `ng_alloc_EB` everywhere, but it's probably a more general question than this QED function.

Reply:
Perfect, that clarifies it. Thanks a lot!

Yes, renaming or just making sure we have the descriptive docstring on our APIs is both good enough. We basically want to avoid that everyone has to check the usage for each parameter in the implementation, when looking at a function interface :)